lib/request-wrapper.ts

```diff
@@ -240,6 +260,7 @@ export class RequestWrapper {
       headers,
       params: qs,
       data,
+      raxConfig: this.retryConfig ?? this.retryConfig,
```

Comment:
I went ahead and looked up the ?? operator :) and I learned that it is called the "nullish coalescing operator" (can't make this up!), but I'm wondering what good does it do to put the same thing on both sides of the ?? operator?  It looks like you'll use `this.retryConfig` if it is defined and non-null and `this.retryConfig` if it null or undefined :)   At least according to my understanding of this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Yes, I should not need the value after `??`.

Reply:
Not sure if this accomplishes what you want, but you could potentially use the "or" operator to make this conditional:
```
raxConfig: this.retryConfig || undefined,
```

All properties that are not set are technically `undefined` so if the `retryConfig` variable isn't set, then the result would be the same as the `raxConfig` not being set at all